### ISSUE TITLE
Closing to settings screen exits the app and not restore the dynamic mode.

### ISSUE THREAD

**Issue by Gniang** [Gniang/RoundedTB#5]

**Comment by Ya-dola** [comment]
Right-clicking the RoundedTB Tray Icon and clicking on 'Hide RoundedTB' works, in 'closing' the settings window while keeping RoundedTB active and allowing for Dynamic Mode to Work.

![image](https://github.com/Gniang/RoundedTB/assets/24356527/d7a230a4-a16c-4092-bc51-ede7f32c848a)

Perhaps the fix is to remap the close button on the settings window to call the function tied to 'Hide RoundedTB'.

Canary Version: RoundedTB_canary20231001

**Comment by Galacticai** [comment]
Same issue

Also the you get `Show RoundedTB` if minimized to task bar, so you have to restore the window, *then* `Hide RoundedTB` will be an option
